apps/site/src/pages/[shortname]/types/entity-type/[...slug-maybe-version].page.tsx

```diff
@@ -65,15 +65,18 @@ const EntityTypePage: NextPage = () => {
       return;
     }
 
-    const schema = getSchemaFromFormData(data);
+    const newLinksAndProperties = getSchemaFromFormData(data);
+
+    const existingSchema = entityType.schema;
 
     const { data: responseData, error: responseError } =
       await apiClient.updateEntityType({
         versionedUri: entityType.schema.$id,
         schema: {
-          ...schema,
-          properties: schema.properties ?? {},
-          title: schema.title ?? entityType.schema.title,
+          ...existingSchema,
```

Comment:
Just in case, do we want to add this?
```suggestion
          ...existingSchema,
          $id: undefined,
```